tests/integration/helpers/helpers.py

```diff
@@ -80,7 +80,10 @@ def get_create2_address(
     """
     return to_checksum_address(
         keccak(
-            b"\xff" + decode_hex(sender_address) + salt + keccak(initialization_code)
+            b"\xff"
+            + to_bytes(hexstr=sender_address)
```

Comment:
I need to investigate more why this change was necessary, but here is some output of my investigations.

I was testing locally before pulling main with

```
def create2_address(caller, bytecode, salt) -> bytes:
    code_hash = keccak(bytes(bytecode))
    data = b"\xff" + to_bytes(caller) + to_bytes(salt) + code_hash
    address = keccak(data)
    return address
```

The `get_create2_address` helper was giving me different results. I inspected it in a debugger session and saw that using `to_bytes` gave equivalent addresses

```
(Pdb) to_checksum_address(keccak(b"\xff" + to_bytes(hexstr=hex(evm_caller_address)) + padded_salt + keccak(bytecode))[-20:])
'0x78EA4300b4e2Ed68ac4BaE23059a825932755c72'
(Pdb) to_checksum_address(expected_create2_addr[-20:])
'0x78EA4300b4e2Ed68ac4BaE23059a825932755c72'
```